synapse/handlers/appservice.py

```diff
@@ -55,6 +55,9 @@ def __init__(self, hs: "HomeServer"):
         self.clock = hs.get_clock()
         self.notify_appservices = hs.config.appservice.notify_appservices
         self.event_sources = hs.get_event_sources()
+        self.msc2409_to_device_messages_enabled = (
```

Comment:
private fields get underscores. Even in classes that do not currently follow the convention.

```suggestion
        self._msc2409_to_device_messages_enabled = (
```